tests/build_tests/super_errors/expected/method_arity_mismatch.res.expected

```diff
@@ -8,5 +8,8 @@
   4 [2m│[0m }
   5 [2m│[0m 
 
-  This function has type (int, int) => unit
-  It is applied with [1;31m1[0m argument but it requires [1;33m2[0m.
\ No newline at end of file
+  This function call is incorrect.
+  The function has type:
```

Comment:
Here the function type is shown at the top, and the explanation of what is wrong is shown below.

In the example further above it is the other way around.

I actually prefer the style here, can we standardize on it?

I.e., always start with

> This function call is incorrect.
> The function has type:
> ...

and then the explanation below?

Or maybe even on one line:

> This function call is incorrect. The function has type:
> ...

Reply:
I adjusted. Please have another look.